deployments/warp_routes/LESS/artheratestnet-holesky-config.yaml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
+  - addressOrDenom: "0x48867511B5c9c6A0E28b36553cd0DC9A0A95A6D2"
+    chainName: artheratestnet
+    collateralAddressOrDenom: "0xcae706200441f8D3f8E880B7C977c57f0c4747d5"
+    connections:
+      - token: ethereum|holesky|0x2201B4cC55c82EE23554C9CDA5CFF62BBa01c69D
+    decimals: 18
+    interchainSecurityModule:
+      modules:
+        - relayer: "0xC34ad0296f606749Ff8F77b75A7d577eB2CDF846"
+          type: trustedRelayerIsm
+        - domains: {}
+          owner: "0xC34ad0296f606749Ff8F77b75A7d577eB2CDF846"
+          type: defaultFallbackRoutingIsm
+      threshold: 1
+      type: staticAggregationIsm
```

Comment:
@bertux the CI is failing because this warp route config has some fields from the warp **deployment** config. The config to deploy a route is different than the config to use the route. This is the config to use it. Did you add these fields yourself? Otherwise it's possible there's a bug in the CLI which we can look into.

Reply:
It does seem likely this is a bug on our end. We'll investigate it today. Once we have it fixed, I'll take over your PR and finish merging it :)

Reply:
I've removed them, it should be found automatically in the registry if I understood well.

Reply:
I've followed the USDC warp route example so I've added the logo of tokens and removed all fields not in the schema.